modules/harvest/src/Commands/HarvestCommands.php

```diff
@@ -126,13 +126,19 @@ protected function buildPlanFromOpts($opts) {
    * @command dkan:harvest:deregister
    */
   public function deregister($id) {
-    try {
-      if ($this->harvestService->deregisterHarvest($id)) {
-        $message = "Successfully deregistered the {$id} harvest.";
+    $this->logger->warning('If you deregister a harvest with published datasets, you will not be able to bulk revert the datasets connected to this harvest.');
```

Comment:
@stefan-korn could you: Split this 159 characters long line (which is greater than 140 authorized).